snyk_threadfix/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.3'
```

Comment:
Rather that hardcoding this in two places, I'd use the https://github.com/rominf/poetry-version library.

Reply:
Nice! Thanks